backend/src/v1/routes/userRoutes.js

```diff
@@ -9,4 +9,6 @@ router.get('/', userController.getAllUsers);
 
 router.post('/', bodyParser.json(), userController.createNewUser);
 
+router.post('/:username', bodyParser.json(), userController.getUser);
```

Comment:
Let's change this to a `GET` request and let's use a query parameter called `user` instead of a route parameter.
https://stackabuse.com/get-query-strings-and-parameters-in-express-js/